packages/validator-query/README.md

```diff
@@ -0,0 +1,57 @@
+# AMP-Toolbox Validator Query
+
+Makes it easier to query published AMP Validator rules and extract information
+about required markup and attributes for all AMP formats.
+
+## Usage
+
+Install via:
+
+```
+$ npm install @ampproject/toolbox-validator-query@canary
+```
+
+### Including the Module
+
+#### ES Module (Browser)
+
+```javascript
+import { load } from '@ampproject/toolbox-validator-query';
+```
+
+#### CommonJs (Node)
+
+```javascript
+const { load } = require('@ampproject/toolbox-validator-query');
+```
+
+### Using the module
+
+```javascript
+  // Loads the validator rules remotely with default options
+  const query = load();
+
+  // Get all tag names used in AMP for Email
+  const names = query.getTagsForFormat('AMP4EMAIL').map(tag => tag.tagName);
```

Comment:
Can you add a comment indicating what the result looks like?

Also: please provide an example for all supported API methods.

Reply:
Linked to relevant proto definitions and added examples for all methods.